Dockerfile

```diff
@@ -29,6 +29,7 @@ WORKDIR /opt/netbox
 RUN pip install -r requirements.txt
 
 COPY docker/configuration.docker.py /opt/netbox/netbox/netbox/configuration.py
+COPY docker/startup_scripts /opt/netbox/netbox/startup_scripts
```

Comment:
There are two ways to use this feature for our users, actually. But in order to be generally useful, having this in the `Dockerfile` seems to be the wrong way.. 

User can either base of the original image and build their own Docker container (see example Dockerfile below), or they can just mount such files into the container using docker-compose (or whatever mechanism; see the example further down).

So, something like the following should probably be added to the README, along with a short explanation of the feature.

```Dockerfile
FROM ninech/netbox
COPY docker/startup_scripts /opt/netbox/netbox/startup_scripts
```

And I think it would be nice to adjust the default `docker-compose.yml` file to include the following:

```
services:
    netbox:
      volumes:
      - ./startup_scripts:/opt/netbox/netbox/startup_scripts
```

This would also require a `.gitkeep` file in the folder `./startup_scripts/`.